rustup-init.sh

```diff
@@ -141,17 +149,17 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        say "\33[1minfo:\33[0m downloading installer"
```

Comment:
You're printing out `rustup - INFO: info:` which is not very ideal...

Two things to be considered in general:
1. Use the normal `info: `, `warn: ` and `error: ` prefixes to align with the binary's logger. Keep the bolded style to make it clearer when ANSI escapes are available, exactly as demonstrated by this `if-else` structure.
2. Some warnings are long and span across several lines.

Reply:
missed that one you're right, thought I had checked all of them 😓 ! 
I also thought that printing `rustup - info` and all was weird, being able to drop the `rustup -` part sounds good to me ^^